src/node/snapchain_node.rs

```diff
@@ -43,10 +44,12 @@ impl SnapchainNode {
 
         let mut consensus_actors = BTreeMap::new();
 
-        let (shard_decision_tx, shard_decision_rx) = mpsc::channel::<Decision>(100);
+        let (shard_decision_tx, shard_decision_rx) = mpsc::channel::<ShardChunk>(100);
 
         let mut shard_messages: HashMap<u32, mpsc::Sender<message::Message>> = HashMap::new();
 
+        let mut shard_stores = HashMap::new();
```

Comment:
Still getting used to rust, but I would have a preference here to annotate the type of this for readability.

- it is being used in a fairly large function with some uses that are far away
- it's not immediately clear what will go in there

I guess my editor can figure it out if I hover over it, but above comments still hold